front/packages/ui/src/details/episode.tsx

```diff
@@ -293,32 +301,56 @@ export const EpisodeLine = ({
 					<Skeleton>
 						{isLoading || (
 							<H6 aria-level={undefined} {...css([{ flexShrink: 1 }, "title"])}>
-								{name ?? t("show.episodeNoMetadata")}
+								{[displayNumber, name ?? t("show.episodeNoMetadata")].join(" · ")}
 							</H6>
 						)}
 					</Skeleton>
-					{isLoading ||
-						(runtime && (
-							<Skeleton>
-								{isLoading || <SubP {...css({ flexShrink: 0 })}>{displayRuntime(runtime)}</SubP>}
-							</Skeleton>
-						))}
+					<View {...css({ flexDirection: "row", alignItems: "center" })}>
+						{isLoading ||
+							(runtime && (
+								<Skeleton {...css(["less"])}>
+									{isLoading || (
+										<SubP>
+											{/* Source https://www.i18next.com/translation-function/formatting#datetime */}
+											{[
+												releaseDate ? t("{{val, datetime}}", { val: releaseDate }) : null,
+												displayRuntime(runtime),
+											]
+												.filter((item) => item != null)
+												.join(" · ")}
+										</SubP>
+									)}
+								</Skeleton>
+							))}
+						{slug && watchedStatus !== undefined && (
+							<EpisodesContext
+								slug={slug}
+								showSlug={showSlug}
+								status={watchedStatus}
+								isOpen={moreOpened}
+								setOpen={(v) => setMoreOpened(v)}
+								{...css([
+									"more",
+									{ display: "flex", marginLeft: ts(3) },
+									Platform.OS === "web" && moreOpened && { display: important("flex") },
+								])}
+							/>
+						)}
+					</View>
 				</View>
 				<View {...css({ flexDirection: "row" })}>
-					<Skeleton>{isLoading || <P numberOfLines={3}>{overview}</P>}</Skeleton>
-					{slug && watchedStatus !== undefined && (
-						<EpisodesContext
-							slug={slug}
-							showSlug={showSlug}
-							status={watchedStatus}
-							isOpen={moreOpened}
-							setOpen={(v) => setMoreOpened(v)}
-							{...css([
-								"more",
-								Platform.OS === "web" && moreOpened && { display: important("flex") },
-							])}
-						/>
-					)}
+					<Skeleton>
+						{isLoading || <P numberOfLines={descriptionExpanded ? undefined : 3}>{overview}</P>}
+					</Skeleton>
+					<IconButton
+						{...css(["more"])}
+						icon={descriptionExpanded ? ExpandLess : ExpandMore}
+						{...tooltip(t(descriptionExpanded ? "misc.collapse" : "misc.expand"))}
+						onPress={(e) => {
+							e.preventDefault();
```

Comment:
Is the preventDefault() needed?

Reply:
Yes it is. It prevents the click event from being passed to the parent `<A>`